src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md

```diff
@@ -119,7 +119,7 @@ On Windows (Powershell):
 ```
 galasactl runs submit local --log - `
 --obr mvn:dev.galasa/dev.galasa.simbank.obr/0.25.0/obr `
---obr mvn:dev.galasa/dev.galasa.uber.obr/0.30.0/obr `
+--obr mvn:dev.galasa/dev.galasa.uber.obr/0.31.0/obr `
```

Comment:
pretty sure you don't need to add this line.

Reply:
thanks - removed